docs/docs/docs/getting-started/installation.md

```
@@ -255,6 +255,23 @@
+
+Use the command `docker compose` command to cleanly shutdown the dev container
+
+```
+docker compose down
+```
+
+#### CLI Startup (Development)
+
+After a successful installation, yse these commands in sequence to start the dev container.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)